src/hotspot/share/gc/shenandoah/shenandoahUtils.hpp

```diff
@@ -42,6 +42,13 @@
 
 class GCTimer;
 
+
+enum StringDedupMode {
```

Comment:
I think the only use is in `ShenandoahMark`, right? Therefore, this declaration should go to `shenandoahMark.hpp`, so we do not have to include `shenandoahUtils.hpp` there?